src/styles/styles.css

```diff
@@ -0,0 +1,132 @@
+:root {
+  --mainFontFamily: "Roboto", sans-serif;
+  --darkColor: #060b35;
+  --mediumColor: #616070;
+  --lightColor: white;
+  --highlightColor: #00acdc;
+  --borderColor: #f3f3f3;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid var(--borderColor);
+  border-radius: 5px;
+
+  background-color: var(--lightColor);
+  font-family: var(--mainFontFamily);
+}
+
+.product-card__image {
+  align-self: center;
+
+  height: 134px;
+  width: 160px;
+
+  margin-bottom: 40px;
+}
+
+.product-card__name {
+  margin-bottom: 4px;
+
+  color: var(--darkColor);
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.product-card__code {
+  margin-bottom: 16px;
+
+  color: var(--mediumColor);
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.product-card__rating {
+  display: flex;
+  justify-content: space-between;
+
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+
+  background-image: url(\images\star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars:last-child {
+  margin-right: 0;
+}
+
+.stars > :nth-child(-n+4) {
+  background-image: url(\images\star-active.svg);
+}
+
```

Comment:
Reuse stars block from and keep it operational. **If you change stars--4 modifier to stars--2 or other - number of selected stars should change correctly.**

Reply:
Thank you! I read this point in the checklist, but didn't really understand:facepalm: